hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusCreateTopicHandler.java

```diff
@@ -61,13 +61,13 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         // The transaction cannot set the admin key unless the transaction was signed by that key
         if (op.hasAdminKey()) {
-            context.requireKeyOrThrow(op.adminKeyOrThrow(), INVALID_ADMIN_KEY);
+            context.requireKeyOrThrow(op.adminKey(), INVALID_ADMIN_KEY);
         }
 
         // If an account is to be used for auto-renewal, then the account must exist and the transaction
         // must be signed with that account's key.
         if (op.hasAutoRenewAccount()) {
-            final var autoRenewAccountID = op.autoRenewAccountOrThrow();
+            final var autoRenewAccountID = op.autoRenewAccount();
```

Comment:
Why are these changes getting rid of the `OrThrow`?

Reply:
I use the next line, context.requireKeyOrThrow(autoRenewAccountID, INVALID_AUTORENEW_ACCOUNT) to capture the PreCheckException and handle it to workflow to handle.